modules/visual_script/visual_script_nodes.cpp

```diff
@@ -2871,9 +2899,13 @@ void VisualScriptCustomNode::_bind_methods() {
 
 	BIND_VMETHOD(MethodInfo(Variant::INT, "_get_input_value_port_type", PropertyInfo(Variant::INT, "idx")));
 	BIND_VMETHOD(MethodInfo(Variant::STRING, "_get_input_value_port_name", PropertyInfo(Variant::INT, "idx")));
+	BIND_VMETHOD(MethodInfo(Variant::INT, "_get_input_value_port_hint", PropertyInfo(Variant::INT, "idx")));
```

Comment:
Public apis should not have _ in front.

Reply:
This is a virtual method intended to overwrite. I think its supposed to be like _this